### PR TITLE
Kubernetes Meetup

### PR DESCRIPTION
**Pull request by gangena**
Please make sure to check the following boxes creating your pull request, thank you very much!

- [x] Add Blog Header, including title, author, image, categories, tags, etc.  

    For example:  

    ```md
    ---
    title: Your Great Blog Title
    author: your_github_id
    categories: project
    tags:
      - python
      - padlocal
    image: /assets/2021/02-your-blog-slug/teaser.jpg
    ---

    One line bio

    Your blog post content Your blog post content
    Your blog post content Your blog post content
    Your blog post content Your blog post content
    ```

    > It will be easier to start drafting your post by using a existing post file as your template, for example, [this one](https://github.com/wechaty/wechaty.js.org/blob/master/jekyll/_posts/2021-01-14-wechaty-puppet-service.md)

- [x] Keep all filenames lowercase, and use `-` to connect words instead of space. e.g. `2017-10-06-wechat-pc-impactor.md` instead of `2017-10-06-WeChat PC Impactor.md`
- [x] Embed video/pdf from YouTube/Bilibili/local (if there's any) by using `{% include iframe.html src="URL_or_PATH" %}`. (Learn more about how to embed file/url in the post by reading this [blog post](https://wechaty.js.org/2020/08/24/add-video-to-wechaty-blog/)
- [x] Save photo to local before publishing, to a folder named by `${MONTH}-${YOUR_BLOG_SLUG}/` under `/assets/2021/` directory.
- [x] Select a beautiful and meaningful teaser image for your blog
- [x] Create your contributor profile (if you are a first time contributor)

    For excample:

    > jekyll/_contributors/lijiarui.md

    ```yaml
    ---
    name: Jiarui LI (李佳芮)
    site: https://lijiarui.github.io
    bio: 句子互动创始人 & CEO，微软人工智能最具价值专家 (AI MVP)
    avatar: /jekyll/assets/contributors/lijiarui/avatar.png
    email: rui@juzi.bot
    twitter: https://weibo.com/u/2175505900
    ---
    ```

  Here is the example file:

  - lijiarui.md: <https://github.com/wechaty/wechaty.js.org/blob/master/jekyll/_contributors/lijiarui.md>
  - It will shown like this on wechaty website: <https://wechaty.js.org/contributors/lijiarui/>

To learn more about the contribute guideline, visit: <https://wechaty.js.org/docs/contributor-program/>

### CODE REVIEW

**Comment by gangena** [comment]
https://github.com/wechaty/wechaty.js.org/issues/909
Corresponding to the first article here

File: jekyll/_posts/2017-11-18-kubernetes-meet-up.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+- 杜卫普：日志易技术总监
+
+十六年运维、安全领域经验，曾任职务Symantec 高级工程师，负责过多家中大型银行、银河证券、移动、中石化等项目，深入了解防火墙、防病毒、漏洞扫描等多种安全产品。
+
+![010](/assets/2017/11-kubernetes-meet-up/010.webp)
+
+- 他带来的话题同样也精彩纷呈，赢来阵阵掌声。
+
+## Wechaty 暑期2020中期路演日
```
Comment (lijiarui) [inline jekyll/_posts/2017-11-18-kubernetes-meet-up.md]: It seems strange here..... What is `Wechaty 暑期2020中期路演日`

**Review by lijiarui — CHANGES_REQUESTED** [review]
http://rui.juzi.bot/presentation/2017-11-17-docker.html

Please change to this blog here.

File: jekyll/_posts/2017-11-18-kubernetes-meet-up.md (hunk 1 above)
Comment (gangena) [reply jekyll/_posts/2017-11-18-kubernetes-meet-up.md]: It has been revised～

Comment (lijiarui) [reply jekyll/_posts/2017-11-18-kubernetes-meet-up.md]: And pay more attention to this:

Please change to this blog here: http://rui.juzi.bot/presentation/2017-11-17-docker.html

**Comment by huan** [comment]
It seems that there's no any Wechaty related content in this event.

**Pull request closed without merging**